package.json

```diff
@@ -23,17 +23,18 @@
     "build": "rimraf dist && run-p build:*",
     "build:js": "rollup -c rollup.config.js",
     "build:types": "tsc --emitDeclarationOnly --declaration",
+    "docs": "typedoc",
     "format": "npm run format:eslint && npm run format:prettier",
-    "format:eslint": "eslint --ext .js,.html . --fix",
-    "format:prettier": "prettier \"**/*.{js,md}\" \"package.json\" --write",
+    "format:eslint": "eslint --ext .ts,.js,.html . --fix",
+    "format:prettier": "prettier \"**/*.{ts,js,md,json}\" --write",
     "lint": "run-p lint:*",
     "lint:eslint": "eslint --ext .ts,.html .",
     "lint:prettier": "prettier \"**/*.ts\" --list-different || (echo '↑↑ these files are not prettier formatted ↑↑' && exit 1)",
     "lint:types": "tsc --noEmit",
     "prepare": "husky install",
     "release": "npm run build && changeset publish"
   },
-  "types": "./dist/types/index.d.ts",
+  "types": "dist/esm/index.d.ts",
```

Comment:
This change is not correct anymore, still should be ./dist/types/index.d.ts

Reply:
It  can also be removed entirely btw, similar to main/module field, as it's covered by exports field